### PR TITLE
fix typos

### PR DESCRIPTION
**Pull request by zaki50**

### CODE REVIEW

**Comment by zaki50** [comment]
@cmelchior please review this or assign appropriate reviewer.

**Comment by emanuelez** [comment]
:+1:

**Comment by zaki50** [comment]
Just information.
All androidTests are GREEN on zaki50_fix_typo branch

**Comment by emanuelez** [comment]
Thank you very much once more @zaki50!! :)

File: realm/src/androidTest/java/io/realm/RealmInterprocessTest.java [hunk 1]
```diff
@@ -185,7 +185,7 @@ private void triggerServiceStep(RemoteProcessService.Step step) {
     }
 
     // Return the service info if it is alive.
-    // When this method return null, it doesn't mean the remote process is not exsited. An 'empty' process could
+    // When this method return null, it doesn't mean the remote process is not existed. An 'empty' process could
```
Comment (bmunkholm) [inline realm/src/androidTest/java/io/realm/RealmInterprocessTest.java]: This sentence has to be fixed "is not existed".

Comment (zaki50) [inline realm/src/androidTest/java/io/realm/RealmInterprocessTest.java]: It should be "is not existing." ?

**Pull request merged**